tizen-api/include/nnstreamer.h

```diff
@@ -227,8 +228,8 @@ int nns_pipeline_construct (const char *pipeline_description, nns_pipeline_h *pi
 int nns_pipeline_destroy (nns_pipeline_h pipe);
 
 /**
- * @brief Get the state of pipeline
- * @detail Get the state of The pipeline handle returned by nns_construct_pipeline (pipe).
+ * @brief Gets the state of pipeline
+ * @detail Gets the state of The pipeline handle returned by nns_construct_pipeline (pipe).
```

Comment:
The -> the

Reply:
nns_constuct_pipeline() or nns_constuct_pipeline (). Which one is right? Are both of them fine?

Reply:
Nice catch! I will.
* The -> the

> nns_constuct_pipeline() or nns_constuct_pipeline (). Which one is right? Are both of them fine?

What about adding space between a name and `()`?
We already have been used this way in our codes.